src/document/DocumentAnnotator.ts

```diff
@@ -10,16 +10,20 @@ import { Mode, setSelectionAction } from '../store';
 import { scrollToLocation } from '../utils/scroll';
 import './DocumentAnnotator.scss';
 
+// Debounce 500ms for keyboard selection
+const SELECTION_CHANGE_DEBOUNCE = 500;
 // Pdf.js textLayer enhancement requires waiting for 300ms (they hardcode this magic number)
 const TEXT_LAYER_ENHANCEMENT = 300;
-// Selection change event is quite noisy, debounce it to avoid unnessary selection changes in store
-const SELECTION_CHANGE_DEBOUNCE = 100;
 
 export default class DocumentAnnotator extends BaseAnnotator {
     annotatedEl?: HTMLElement;
 
+    isMouseSelecting = false;
```

Comment:
Ideally, all state should live in the redux store. What is the reasoning for keeping this as an instance value?

Reply:
The store instance and annotator instance create and destroy together, so I think an instance variable in annotator is equivalent to a state in store and the variable is only used in this class.